src/plotly/plotly_types.nim

```diff
@@ -136,6 +148,7 @@ type
     hovermode*: HoverMode
     autosize*: bool
     showlegend*: bool
+    rangeslider*: bool
```

Comment:
can you add a doc here what this does and how it's different from the Axis rangeslider? I need to get better about documenting these attributes as well.

Reply:
That is spurious (from a previous failed attempt). Well spotted I'll remove it.